sor/src/test/java/com/bazaarvoice/emodb/sor/core/RedundantDeltaTest.java

```diff
@@ -480,7 +481,12 @@ private Audit newAudit(String comment) {
                 build();
     }
 
-    private List<UUID> toChangeIds(Iterator<Map.Entry<UUID, Change>> iterator) {
+    private List<UUID> toChangeIds(Iterator<Map.Entry<DeltaClusteringKey, Change>> iterator) {
+        return ImmutableList.copyOf(
+                Iterators.transform(iterator, entry -> entry.getKey().getChangeId()));
+    }
+
+    private List<DeltaClusteringKey> toClusteringKeys(Iterator<Map.Entry<DeltaClusteringKey, Change>> iterator) {
         return ImmutableList.copyOf(
                 Iterators.transform(iterator, entry -> entry.getKey()));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AS1612)